lib/bundler/cli/plugin.rb

```diff
@@ -19,5 +19,10 @@ class CLI::Plugin < Thor
     def install(*plugins)
       Bundler::Plugin.install(plugins, options)
     end
+
+    desc "uninstall PLUGINS", "Uninstall the plugins"
```

Comment:
@hmistry , the command will be 
`bundle plugin uninstall plugin_name1 plugin_name2 ` . Right now it is not installing anything if `bundle plugin uninstall` is executed. I think there must be an option `--all` to uninstall all the plugins. I will edit the PR accordingly.